### PR TITLE
Fix "app not starting" + W.I.P. Annotators + fix tests

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [ ] Unit tests
- [ ] Updated testplan

### CODE REVIEW

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/filter/Location.java [hunk 1]
```diff
@@ -17,10 +17,10 @@
 {
 	public abstract String getChrom();
 
-	public abstract long getPos();
+	public abstract int getPos();
```
Comment (Mark-de-Haan) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/filter/Location.java]: Why the type change?

File: molgenis-data/src/main/java/org/molgenis/data/meta/model/AttributeMetaData.java [hunk 2]
```diff
@@ -536,4 +536,6 @@ private static String toEnumOptionsString(List<String> enumOptions)
 	{
 		return !enumOptions.isEmpty() ? enumOptions.stream().collect(joining(",")) : null;
 	}
+
+
```
Comment (Mark-de-Haan) [inline molgenis-data/src/main/java/org/molgenis/data/meta/model/AttributeMetaData.java]: No changes in this file

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/AnnotatorConfig.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package org.molgenis.data.annotation.entity;
+
+/**
+ * Created by charbonb on 24/06/16.
+ */
```
Comment (Mark-de-Haan) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/entity/AnnotatorConfig.java]: IntelliJ comments

File: molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/filter/Location.java (hunk 1 above)
Comment (bartcharbon) [inline molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/filter/Location.java]: because the type was wrong

**Pull request merged**